README.zh.md

```diff
@@ -61,22 +62,21 @@ OpenYurt 遵循经典的云边一体化架构。
 安装OpenYurt前，请检查[资源和系统要求](./docs/resource-and-system-requirements-cn.md)
 
 ## 开始使用
-OpenYurt 支持最高版本为1.21的 Kubernetes 。使用更高版本的 Kubernetes 可能会导致兼容性问题。
+OpenYurt 支持最高版本为1.22的 Kubernetes 。使用更高版本的 Kubernetes 可能会导致兼容性问题。
 
 为了用户更方便的使用OpenYurt，用户可以根据下表，选择合适的方法来安装OpenYurt。
 
 | 体验方法                                 | 参考链接                                                     | 预计时间 |
 | ---------------------------------------- | ---------------------- | -------- |
 | 通过OpenYurt体验中心 | [OpenYurt体验中心](https://openyurt.io/docs/next/installation/openyurt-experience-center/overview) | < 1min   |
-| 从零安装一个OpenYurt集群                 | [yurtctl](https://openyurt.io/docs/next/installation/yurtctl-init-join) | <5min    |
-| 通过Operator把K8s集群转换为OpenYurt集群        | [yurtcluster-operator](https://openyurt.io/docs/next/installation/yurtcluster)       | <5min    |
-| 手动转换K8s集群为OpenYurt集群            | [manual](https://openyurt.io/docs/next/installation/manually-setup)    | 10min   |
+| 从零安装一个OpenYurt集群                 | [yurtadm init/join](https://openyurt.io/zh/docs/next/installation/yurtadm-init-join) | <5min    |
+| 基于Kubernetes手动安装OpenYurt集群            | [manual](https://openyurt.io/docs/next/installation/manually-setup)    | 10min   |
```

Comment:
We should add a kubernetes prepare step. And review the pr `https://github.com/openyurtio/openyurt.io/pull/109`.

Reply:
fixed